integration-tests/contracts/contract_models.go

```diff
@@ -134,6 +137,7 @@ type OffchainAggregator interface {
 	Fund(nativeAmount *big.Float) error
 	GetContractData(ctx context.Context) (*OffchainAggregatorData, error)
 	SetConfig(chainlinkNodes []*client.Chainlink, ocrConfig OffChainAggregatorConfig) error
```

Comment:
These two methods feel like they can be merged into one that takes list of senders instead of pulling them from the node directly